torch/csrc/api/src/nn/module.cpp

```diff
@@ -76,6 +76,7 @@ const std::string& Module::name() const noexcept {
 }
 
 std::shared_ptr<Module> Module::clone(const optional<Device>& device) const {
+  (void)device;
```

Comment:
why do we need it?

Reply:
Clang Analyzer gives the unused variable warnings and this silences it. Thought it would make the code cleaner. Should I remove these?

Reply:
Yes I think we should remove them. We can probably deal with the unused variable warnings in other ways in the future.